commands/mute/tempmute.js

```diff
@@ -0,0 +1,127 @@
+
+            channel.send(tempmuteEmbed);
+
+        const tempUnMuteEmbed = new Discord.MessageEmbed()
+            .setColor('#0099ff')
+            .setTitle(`${toTempMute.user.username}#${toTempMute.user.discriminator} was unmuted after ${lengthOfTime}:`)
+            .setThumbnail(`https://cdn.discordapp.com/avatars/${toTempMute.user.id}/${toTempMute.user.avatar}.png`)
+            .setTimestamp()
+            .setFooter(`${msg.guild.name}`);
+
+        // Add record to infractions table.
+        let now = new Date();
+        let timestamp = dateFormat(now, "yyyy-mm-dd HH:MM:ss");
+
+        var sqlInfractions = `INSERT INTO infractions (timestamp, user, action, length_of_time, reason, valid, moderator) 
+        VALUES ('${timestamp}', '${toTempMute}', 'cc!tempmute', '${lengthOfTime}', '${reason}', true, '${msg.author.tag}')`;
+
+        con.query(sqlInfractions, function (err, result) {
+            if (err) {
+            console.log(err);
+            } else {
+            console.log("1 record inserted into infractions.");
+            }
+        });
+
+        var sqlModLog = `INSERT INTO mod_log (timestamp, moderator, action, length_of_time, reason) 
+        VALUES ('${timestamp}', '${msg.author.tag}', '${msg}', '${lengthOfTime}', '${reason}')`;
+
+        con.query(sqlModLog, function (err, result) {
+            if (err) {
+            console.log(err);
+            } else {
+            console.log("1 record inserted into mod_log.");
+            }
+        });
+
+        const unmute = (message) => {
+            toUnmute = message.mentions.members.first();
+            toUnmute.roles.remove(message.guild.roles.cache.find(role => role.name === "Muted"));
+            message.channel.send(`${toUnmute} was unmuted.`);
+            toUnmute.send("You've been unmuted.");
+        }
+
+        timestamp = dateFormat(now, "yyyy-mm-dd HH:MM:ss");
+
+        var sqlInfractions2 = `INSERT INTO infractions (timestamp, user, action, length_of_time, reason, valid, moderator) 
+        VALUES ('${timestamp}', '${toTempMute}', 'cc!unmute', 'N/A', 'tempmute expired', true, 'automatic')`;
+
+        var sqlModLog2 = `INSERT INTO mod_log (timestamp, moderator, action, length_of_time, reason) 
+        VALUES ('${timestamp}', 'automatic', 'cc!unmute', 'N/A', 'tempmute expired')`;
+
+        setTimeout(() => {
+            unmute(msg)
+            channel.send(tempUnMuteEmbed)
+            con.query(sqlInfractions2, function (err, result) {
+                if (err) {
+                console.log(err);
+                } else {
+                console.log("1 record inserted into infractions.");
+                }
+            })
+            con.query(sqlModLog2, function (err, result) {
+                if (err) {
+                console.log(err);
+                } else {
+                console.log("1 record inserted into mod_log.");
+                }
+            });    
+        }, ms(lengthOfTime));
```

Comment:
Same thing... might want to put this all on one line

Reply:
Done